src/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const INTERNAL_PREFIX = "/_frsh";
 export const DEV_ERROR_OVERLAY_URL = `${INTERNAL_PREFIX}/error_overlay`;
 export const ALIVE_URL = `${INTERNAL_PREFIX}/alive`;
+export const PARTIAL_SEARCH_PARAM = "fresh-partial";
```

Comment:
Files in `src/runtime` already imported from this file, so it seemed reasonable to move this constant here so we could avoid it being hard-coded in `src/context.ts`.